docker/build-tools/Dockerfile

```
@@ -52,7 +52,7 @@ ENV GOLANG_PROTOBUF_VERSION=v1.33.0
 ENV GO_BINDATA_VERSION=v3.1.2
 ENV GO_JUNIT_REPORT_VERSION=df0ed838addb0fa189c4d76ad4657f6007a5811c
 ENV HADOLINT_VERSION=v2.12.0
-ENV HELM3_VERSION=v3.14.2
+ENV HELM3_VERSION=v3.14.3
 ENV HUGO_VERSION=0.123.8
```

Not sure if we wait for a .1, but
```suggestion
ENV HUGO_VERSION=0.124.0
```